src/main/java/racingcar/constant/ErrorMessage.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.constant;
+
+public enum ErrorMessage {
+    FIVE("5"),
+    TWO("2"),
+    ZERO("0"),
```

Comment:
개인적인 의견으로는 확장성을 위해서 상수를 두는 것이라면
자동차 이름의 Max 길이라는 것을 암시하는 이름과 "5"를 두는게 
좋지 않았을까 라는 생각이 들어요!

왜냐면 이후에 6으로 변경이 된다면 상수화 했음에도 결국 SIX("6") 그리고 밑에 OUT_OF_STANDARD에서도 SIX로 변경하셔야 하니깐요!
개인적인 의견입니다!!  어떻게 생각하실까요! 🥰

Reply:
200% 동의합니다! 사실 이 부분은 **제출 10분 전에 떠올린 코드** 에요..😭
급한 마음에 `상수로 치환`하는 것에만 신경을 썼다가, 12시 지나고 생각이 났어요 ㅋㅋㅋㅋㅋㅋ

 ```suggestion
    MAX_STANDARD("5"),
    NOT_SINGLE_VALUE("2"),
    MIN_STANDARD("0"),
```

로 했다면 좋았을 것을..